README.md

```diff
@@ -1,15 +1,55 @@
-# feral
+# feral [![Discord](https://img.shields.io/discord/632277896739946517.svg?label=&logo=discord&logoColor=ffffff&color=404244&labelColor=6A7EC2)](https://discord.gg/AJASeCq8gN)
 
-To learn more please read the ["Serverless Concept" gist](https://gist.github.com/djspiewak/37a4ea0d7a5237144ec8b56a76ed080d).
+feral is a framework for writing serverless functions in Scala with [Cats Effect](https://github.com/typelevel/cats-effect) and deploying them to the cloud, targeting both JVM and JavaScript runtimes. Integrations with libraries such as [circe](https://github.com/circe/circe) and [http4s](https://github.com/http4s/http4s) enable feral to provide an idiomatic, purely functional interface that is flexible and composable. The initial focus has been on supporting [AWS Lambda](https://aws.amazon.com/lambda/) and will eventually expand to other serverless providers.
 
-## Usage
+## Getting started
 
-```sbt
-libraryDependencies += "org.typelevel" %% "feral" % "<version>"
-```
-
-Published for Scala 2.13.6.
+Feral is published for Scala 2.13 and 3.1+ with artifacts for both JVM and Scala.js 1.8+.
 
 ```scala
-import feral._
+// Everything you need to write your AWS Lambda
+libraryDependencies += "org.typelevel" %%% "feral-lambda" % "0.1.0-M1"
+
+// Specialized integrations
+libraryDependencies += "org.typelevel" %%% "feral-lambda-http4s" % "0.1.0-M1"
+libraryDependencies += "org.typelevel" %%% "feral-lambda-cloudformation-custom-resource" % "0.1.0-M1"
 ```
+
+`IOLambda[Event, Result]` or `IOLambda.Simple[Event, Result]` are the entrypoint for your Lambda (comparable to `IOApp` in Cats Effect). `Event` and `Result` can be any type for which a circe `Decoder` and `Encoder` are defined and must correspond to the "trigger" event and (optional) result for your Lambda. The `feral.lambda.events` package provides some of these models already ([please consider contributing!](https://github.com/typelevel/feral/issues/48)).
+
+The [examples](https://github.com/typelevel/feral/tree/main/examples/src/main/scala/feral/examples) provide an in-depth demonstration of how to implement your Lambda.
```

Comment:
Not really in-depth at this moment :)

Reply:
Haha, good point. I'm planning to take a pass through and flesh them out a bit, but even after that I suspect this will be an exaggeration :P